main.go

```diff
@@ -288,27 +332,27 @@ func (f *_escFile) Sys() interface{} {
 	return f
 }
 
-// FS returns a http.Filesystem for the embedded assets. If useLocal is true,
+// {{.FunctionPrefix}}{{.FSFunctionName}} returns a http.Filesystem for the embedded assets. If useLocal is true,
```

Comment:
I'd prefer to hard code FSFunctionName, DirFunctionName, etc. Only leave as parameters things that can change (currently just FunctionPrefix and PackageName).